android/app/src/main/java/app/covidshield/receiver/worker/ExposureCheckSchedulerWorker.kt

```diff
@@ -33,7 +43,17 @@ class ExposureCheckSchedulerWorker (val context: Context, parameters: WorkerPara
             UniqueDailyDebugMetricsHelper.markMetricAsPublished("200.0", context)
         }
 
-        MetricsService.publishDebugMetric(2.0, context)
+        try {
+            val status = storageService.retrieve(StorageDirectory.ExposureStatus)
+            val exposureStatus = Gson().fromJson(status, ExposureStatus::class.java)
+            val minutesSinceLastCheck = (DateFns.getCurrentDate() - exposureStatus.lastChecked.timestamp) / 1000 / 60
```

Comment:
The implementation of `DateFns.getCurrentDate` (System.currentTimeMillis) will provide UTC timestamp but I think we save a Local timestamp on the React side (getCurrentDate).